corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py

```diff
@@ -17,11 +17,30 @@ class Command(ElasticReindexer):
 
     sort_key_include_docs = True
 
-    def get_extra_view_kwargs(self):
-        return {
-            'startkey': ['XFormInstance'],
-            'endkey': ['XFormInstance', {}],
-        }
+    def full_couch_view_iter(self):
+        # copied from couchforms/_design/filters/xforms.js
+        doc_types = [
+            'XFormInstance',
+            'XFormArchived',
+            'XFormError',
+            'XFormDeprecated',
+            'XFormDuplicate',
+            'XFormInstance-Deleted',
+            'HQSubmission',
+            'SubmissionErrorLog',
```

Comment:
@dannyroberts From quick glance at the ticket, it looks addionally `XFormDuplicate` docs are to be indexed. Curious why you added other doc_types (for e.g. `XFormInstance-Deleted`) to indexer.